dataplane/vppagent/cmd/vppagent-dataplane.go

```diff
@@ -37,8 +37,7 @@ func main() {
 
 	registration := common.CreateDataplane(agent, dataplaneProbes)
 
-	select {
-	case <-c:
+	for range c {
```

Comment:
Did this faster?

Reply:
Not sure which is faster, but this is fixed due to an error raised by the linter checker, i.e. single channels should not use select.